data/models/noteModel.js

```diff
@@ -0,0 +1,138 @@
+const db = require('../dbConfig.js');
+
+const get = async (id) => {
+    if (id) {
+        const note = await db('notes').where('id', id);
```

Comment:
I'm pretty sure you can use `.first()`? instead of having that if statement
```suggestion
        const note = await db('notes').where('id', id).first();
```